Dockerfile

```diff
@@ -62,6 +66,9 @@ ADD autograph.yaml /app
 ADD version.json /app
 COPY --from=builder /go/bin /go/bin/
 
+# Copy Google KMS library from the builder.
+COPY --from=builder /tmp/libkmsp11.so /app
```

Comment:
What directory does the .so need to be in to be automatically picked up. `/usr/local/lib` maybe?

Reply:
As I understand things, it will never be automatically picked up. We need to explicitly specify the path to it in `hsm.path` in the app config. Via [a](https://github.com/mozilla-services/autograph/blob/057f7ee6bb490bc6e5d5fbdd7536c43d943957e3/main.go#L380) [bit](https://github.com/ThalesGroup/crypto11/blob/c73933259cb60509d00f32306eea53d10f8e8f10/crypto11.go#L239) [of](https://github.com/miekg/pkcs11/blob/9078ad6b9d4b17a548e56959b10e8949afab10ec/pkcs11.go#L789-L791) [indirection](https://github.com/miekg/pkcs11/blob/9078ad6b9d4b17a548e56959b10e8949afab10ec/pkcs11.go#L74-L91) we end up calling `dlopen` and have an indirect handle on the loaded library.